packages/swap-manager/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "@summerfi/typescript-config/tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "dist",
+    "baseUrl": ".",
+    "paths": {
+      "~swap-manager": ["./src/index.ts"],
+      "~swap-manager/*": ["./src/*"],
+      "@summerfi/sdk/*": ["../sdk/src/*"],
+      "~sdk/*": ["../sdk/src/*"]
```

Comment:
It's an antipattern. We should not include files not from the `src` folder.

Reply:
Agreed. As discussed offline, Piter has found a compromise solution. Please check it when you can to see what do you think about it